trace-normalization/src/normalize_utils.rs

```diff
@@ -1,332 +1,367 @@
-// Copyright 2023-Present Datadog, Inc. https://www.datadoghq.com/
+// Copyright 2024-Present Datadog, Inc. https://www.datadoghq.com/
 // SPDX-License-Identifier: Apache-2.0
 
-// DEFAULT_SERVICE_NAME is the default name we assign a service if it's missing and we have no
-// reasonable fallback
-const DEFAULT_SERVICE_NAME: &str = "unnamed-service";
-
-// MAX_NAME_LEN the maximum length a name can have
+use std::time::SystemTime;
+
+// MAX_TYPE_LEN the maximum size for a span type
+pub(crate) const MAX_TYPE_LEN: usize = 100;
+/// an arbitrary cutoff to spot weird-looking values
+/// nanoseconds since epoch on Jan 1, 2000
+const YEAR_2000_NANOSEC_TS: i64 = 946684800000000000;
+/// DEFAULT_SPAN_NAME is the default name we assign a span if it's missing and we have no reasonable
+/// fallback
+pub(crate) const DEFAULT_SPAN_NAME: &str = "unnamed_operation";
```

Comment:
Where does this come from? I found this ancient [RFC](https://github.com/Datadog/architecture/blob/master/rfcs/apm/agent/trace-agent-normalization/rfc.md) saying `unnamed-operation`.

Reply:
The [agent](https://github.com/DataDog/datadog-agent/blob/c746874a5c0e7c1745a5d10d87d0847721fd54b1/pkg/trace/traceutil/normalize.go#L18) and most [tracers](https://github.com/search?q=org%3ADataDog+unnamed+operation&type=code) seem to be using `unnamed_operation`. Although I agree that `unnamed-operation` makes more sense.

Reply:
Let's keep doing what we have been doing, and use `unnamed_operation`.